src/index.html

```diff
@@ -22,6 +22,423 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="header__link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
```

Comment:
The class 'is-active' should be 'nav__link--active' to follow BEM methodology as per the code style rules. Modifier names should be formed as block-name__element-name--modifier-name.